src/lib/sidenav/sidenav.ts

```diff
@@ -384,8 +380,12 @@ export class MdSidenavContainer implements AfterContentInit {
     });
     this._validateDrawers();
 
-    // Give the view a chance to render the initial state, then enable transitions.
-    first.call(this._ngZone.onMicrotaskEmpty).subscribe(() => this._enableTransitions = true);
+    // Give the view a chance to render the initial state, then enable transitions. Note that we
+    // don't use data binding, because we're not guaranteed that newer version of Angular will
+    // re-evaluate them after we set the flag here.
+    first.call(this._ngZone.onMicrotaskEmpty).subscribe(() => {
```

Comment:
The main point of this is to defer the animation until the next change detection cycle, because we don't want sidenavs to animate in on load. Setting the flag in `AfterViewInit` is still a little too early.